test/new_relic/agent/aws_test.rb

```diff
@@ -6,18 +6,26 @@
 
 class AwsTest < Minitest::Test
   def test_create_arn
```

Comment:
Do you think it'd be beneficial to add tests that call some of the different AWS classes related to retrieving credentials? I wonder if that'd give us a heads up if a future version breaks our ARN generation.

Reply:
Yeah, unfortunately we could only do that in multiverse tests, so we'd have to move these into a multiverse suite. Also, I don't think we could use the one that was broken since, for that one, you have to be running in EC2 to be able to actually use it. Also, since the responses are stubbed for our dynamodb tests, there are no credentials being created anyways. So we'd need to figure out a way around those things. i'd be down to chat about it later and come up with some ideas if you want

Reply:
We chatted about this sync and decided to leave the tests as they are.